homeassistant/components/lutron_caseta/light.py

```diff
@@ -48,37 +57,164 @@ async def async_setup_entry(
 
 
 class LutronCasetaLight(LutronCasetaDeviceUpdatableEntity, LightEntity):
-    """Representation of a Lutron Light, including dimmable."""
+    """Representation of a Lutron Light, including dimmable, white tune, and spectrum tune."""
 
-    _attr_color_mode = ColorMode.BRIGHTNESS
-    _attr_supported_color_modes = {ColorMode.BRIGHTNESS}
     _attr_supported_features = LightEntityFeature.TRANSITION
 
+    SUPPORTED_COLOR_MODE_DICT = {
+        DEVICE_TYPE_SPECTRUM_TUNE: {
+            ColorMode.HS,
+            ColorMode.COLOR_TEMP,
+            ColorMode.WHITE,
+        },
+        DEVICE_TYPE_WHITE_TUNE: {ColorMode.COLOR_TEMP},
+    }
```

Comment:
Please make this a constant instead of binding it to the class since we don't need to override it in children

Reply:
Should I put it in the const file or just define it at the top of the file?

Reply:
Define at the top please since its light platform specific